modules/tinymce/src/plugins/link/main/ts/api/Commands.ts

```diff
@@ -1,17 +1,20 @@
 import Editor from 'tinymce/core/api/Editor';
 
-import * as Actions from '../core/Actions';
+import * as Dialog from '../ui/Dialog';
 import * as Options from './Options';
 
+interface Dialog {
```

Comment:
This name conflicts with the import above and it's not related to the dialog anyways, it's the command options. So maybe something like `CommandOptions` instead?

Reply:
Ah that explains the failures haha, thanks @lnewson